extract.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -uevo pipefail
+set -evo
```

Comment:
`o` is for `option` and supposes to use argument, like `pipefail`. If there are no arguments `o` is unnecessary.